app/routes/products.js

```diff
@@ -29,4 +29,15 @@ router.post('/', (req, res) => {
     res.status(200).send();
 	});
 });
+
+router.get('/:id', (req, res) => {
+  Product.find({'_id': req.params.id}, (err, product) => {
```

Comment:
We only want a single object as a result, not an array.  Should be using `Product.findOne(...)`